server/api/constants.go

```diff
@@ -46,3 +49,13 @@ const (
 	PropChannelID = "channel_id"
 	PropPostID    = "post_id"
 )
+
+// KV namespace
+const (
+	KeyCallOnce     = "CallOnce"
```

Comment:
Not sure what will be finally the keys for apps KV values, but does it make sense to add a prefix to these? Something like "mmapps_"? That way it is easier to notify and check whether an app key is valid (if we form them as appid_key we just have to make sure the app_id != mmapps)